controllers/platformnetwork_controller.go

```diff
@@ -71,25 +71,29 @@ func (r *PlatformNetworkReconciler) UpdateInsyncStatus(client *gophercloud.Servi
 // Reconciled/Insync to false only if Generation != ObservedGeneration.
 // Note that for deletion we are just cleaning up the finalizer and we
 // are not specifically deleting network object on the system.
-func (r *PlatformNetworkReconciler) ReconcileResource(client *gophercloud.ServiceClient, instance *starlingxv1.PlatformNetwork, request_namespace string) (err error) {
+func (r *PlatformNetworkReconciler) ReconcileResource(client *gophercloud.ServiceClient, instance *starlingxv1.PlatformNetwork, request_namespace string, scope_updated bool) (err error) {
 	if instance.DeletionTimestamp.IsZero() {
-		if instance.Status.ObservedGeneration != instance.ObjectMeta.Generation {
+		if instance.Status.ObservedGeneration != instance.ObjectMeta.Generation ||
+			scope_updated {
 			host_instance, err := r.CloudManager.GetActiveHost(request_namespace, client)
 			if err != nil {
 				msg := "failed to get active host"
 				logPlatformNetwork.Error(err, msg)
 				return common.NewUserDataError(msg)
 			}
 
-			host_instance.Status.InSync = false
-			host_instance.Status.Reconciled = false
-			err = r.Client.Status().Update(context.TODO(), host_instance)
+			err = r.CloudManager.NotifyResource(host_instance)
 			if err != nil {
-				msg := fmt.Sprintf("Failed to update '%s' host instance status", host_instance.Name)
+				msg := fmt.Sprintf("Failed to notify '%s' active host instance", host_instance.Name)
 				logPlatformNetwork.Error(err, msg)
 				return common.NewResourceConfigurationDependency(msg)
 			}
 
+			r.ReconcilerEventLogger.NormalEvent(instance,
+				common.ResourceUpdated,
```

Comment:
As we are not updating the host, can we use another event?

Reply:
Done, created `common.ResourceNotified` event.